### PR TITLE
Add Attachment API object support

### PR DESCRIPTION
**Pull request by dmitry-ilyashevich**

### CODE REVIEW

File: lib/rubill/session.rb [hunk 1]
```diff
@@ -67,8 +67,19 @@ def _post(url, data, retries=0)
...
+      if options.key?(:fileName)
+        file = StringIO.new(options.delete(:content))
+      end
+
+      post_options = {
+        body: options,
+        headers: default_headers,
+      }
+
+      post_options[:file] = file if file
```
Comment (ataber) [inline lib/rubill/session.rb]: Correct me if I'm wrong, but won't this throw a `NameError` if file has not been defined on line 71?

Comment (dmitry-ilyashevich) [inline lib/rubill/session.rb]: You can test it yourself by trying this code snippet in your console:

    2.2.1 :002 > options = {}
     => {}
    2.2.1 :003 >       if options.key?(:fileName)
    2.2.1 :004?>           file = StringIO.new(options.delete(:content))
    2.2.1 :005?>         end
     => nil
    2.2.1 :006 >   file
     => nil

Or even simpler:

    if false
        foo = 1
    end

    foo

name defines in block, not in a condition.

Comment (ataber) [inline lib/rubill/session.rb]: Sounds good!

**Comment by ataber** [comment]
Looks good to me! Just resolve the conflicts and I'll happily merge

**Pull request merged**